packages/ui/libs/device/data/src/fetching/deviceFirstImage.ts

```diff
@@ -9,6 +9,8 @@ export const useDeviceFirstImageUrl = (
   const [imageUrl, setImageUrl] = useState('');
 
   const getAndSetImage = async (id: string) => {
+    if (id === 'null') return;
+
     const response = await publicFileDownloadHandler(id);
```

Comment:
Looks a bit strange. Is it really `"null"` string? If yes, maybe it is better to fix somewhere before this happens
The same for `packages/ui/libs/device/data/src/fetching/deviceImages.ts:11`

Reply:
I agree, this is how i received it from `/api/device-registry` GET:
![Screenshot 2021-11-25 at 09 59 13](https://user-images.githubusercontent.com/69903849/143402238-b80ba720-7ded-4392-9e4b-d973faaf9aec.png)

Reply:
So this is definitely a bug, I'll check what is going on there

Reply:
it was a bug on the backend, added a task to validate imageIds.